src/handler/http/models/dashboards.rs

```diff
@@ -0,0 +1,228 @@
+use config::meta::dashboards::{v1, v2, v3, v4, v5};
+use serde::{Deserialize, Serialize};
+use serde_json::Value as JsonValue;
+use utoipa::ToSchema;
+
+/// HTTP request body for the `CreateDashboard` endpoint.
+#[derive(Debug, Deserialize, ToSchema)]
+pub struct CreateDashboardRequestBody(JsonValue);
```

Comment:
The schema of request values doesn't follow a pattern that can be easily expressed using Serde and structs. Therefore, for now at least, I'm just wrapping each Request Body struct around a JSON value and then doing more complicated version-based parsing of that JSON value in the transformation functions from the "HTTP model" to the "Service Layer" model.

Reply:
One small benefit that we have here, is that we can at least use Serde and Actix to validate that the request body is valid JSON for us before we start processing the request body